lib/Magic.js

```diff
@@ -32,6 +31,9 @@ class Magic {
             if (path.indexOf(sep) === 0) {
                 path = path.substring(sep.length);
             }
+
+            if(ignoreSeparator)
```

Comment:
add a space between the if and bracket. `if (ignoreSeparator)`

Reply:
done